Feature/format scraped data

Araujoch opened the pull request:
Applies manual filtering and enforces a language confidence score threshold (≥ 0.70).

Commit:
Add 18 extracted JSONL files. #8

Commit:
Add 35 extracted JSONL files. #8

Commit:
add document normalization and report generation

Commit:
Add 0.70 language score filtering and manual filtering

joausaga commented on src/guarascraped/README.md:
```diff
@@ -0,0 +1,134 @@
+
+------------------------------------------------------------------------
+
+## 📁 Dataset Overview
```
The title should be `## Downloaded Data`

joausaga commented on src/guarascraped/README.md:
````diff
@@ -0,0 +1,134 @@
+}
+```
+
+------------------------------------------------------------------------
````
Before this, we should describe the processing procedure, pointing the reader to the directory where the processing results are stored

joausaga commented on src/guarascraped/README.md:
```diff
@@ -0,0 +1,134 @@
+
+### 📌 Documents
+
+-   **Total extracted documents:** 61167
```
This is different from what is reported in `processed/all_domains_report.json`. All statistics should be consistent with what is reported there

joausaga commented on src/guarascraped/README.md:
```diff
@@ -0,0 +1,134 @@
+
+------------------------------------------------------------------------
+
+## 🧩 Processing Pipeline (Normalization + Report Generation)
```
Anticipate this

joausaga commented on src/guarascraped/README.md:
```diff
@@ -0,0 +1,134 @@
+
+------------------------------------------------------------------------
+
+## ⚙️ Script Used
```
Anticipate this. In general, first we tell the method then we describe the results

joausaga commented on src/guarascraped/README.md:
```diff
@@ -0,0 +1,134 @@
+
+------------------------------------------------------------------------
+
+## ▶️ How to Run the Script
```
Anticipate this

joausaga commented on src/guarascraped/README.md:
```diff
@@ -0,0 +1,134 @@
+
+## 📝 Note
+
+This dataset contains **only the raw scraping output**. It may include
```
This is not totally true. Here we also have processed text.

joausaga commented on src/guarascraped/formart_data.py:
```diff
@@ -0,0 +1,234 @@
+FINAL_JSONL = os.path.join(OUTPUT_DIR, "all_domains.jsonl")
+FINAL_REPORT = os.path.join(OUTPUT_DIR, "all_domains_report.json")
+
+os.makedirs(OUTPUT_DIR, exist_ok=True)
```
Put this inside the main

joausaga commented on src/guarascraped/formart_data.py:
```diff
@@ -0,0 +1,234 @@
+word_seg = spacy.blank("xx")
+
+# identificador de lenguaje
+identifier = LanguageIdentifier(glotlid=True, fasttext=True, openlid=True)
```
The language identifier object should be instantiate inside the main

joausaga commented on src/guarascraped/formart_data.py:
```diff
@@ -0,0 +1,234 @@
+
+
+# ============================
+# PROCESO PRINCIPAL
```
Comment in English

joausaga commented on src/guarascraped/formart_data.py:
```diff
@@ -0,0 +1,234 @@
+# ============================
+
+def main():
+    print(f"{BLUE}=== Procesando TODOS los dominios ==={RESET}")
```
Informative messages in English

joausaga commented on src/guarascraped/formart_data.py:
```diff
@@ -0,0 +1,234 @@
+# ============================
+
+
+BASE_DIR = Path(__file__).resolve().parent.parent.parent
```
Why do we have these constant definitions here and not inside the main function? They are not used anywhere apart from the main.

joausaga commented on src/guarascraped/formart_data.py:
```diff
@@ -0,0 +1,234 @@
+    files.sort()
+    num_domains = len(files)
+
+    # contar documentos totales
```
Comment in English

joausaga reviewed (changes requested):
The code needs further polishing and improvement before being ready to be merged

Commit:
Correct README scope and refactor processing script according to review comments

Araujoch commented:
Hello, thank you for the review.

I have addressed all the requested changes:
- I have updated the README file.

- I have refactored the processing script based on the review comments.

If you need any further adjustments, please let me know.

joausaga commented on src/guarascraped/README.md:
```diff
@@ -1,134 +1,132 @@
-Each `.jsonl` file contains the documents extracted from that site, with
-objects such as:
+The raw scraping output was **normalized and analyzed** using a processing
+pipeline based on the methodology of the  
```
Better 

"The raw scraping output was **normalized and analyzed** using a processing methodology implemented in [link_to_repo]"

joausaga commented on src/guarascraped/README.md:
```diff
@@ -1,134 +1,132 @@
+The processing pipeline includes:
 
-------------------------------------------------------------------------
+- document-level metadata normalization
```
What we mean by document-level metadata normalization needs further explanation. If it is explained later, say it here.

joausaga commented on src/guarascraped/README.md:
```diff
@@ -1,134 +1,132 @@
-------------------------------------------------------------------------
+- document-level metadata normalization
+- character and word counts using `split()` and spaCy
+- language identification and scoring
```
How do we do language identification and scoring?

joausaga commented on src/guarascraped/README.md:
````diff
@@ -1,134 +1,132 @@
+```
 
--   **Total extracted documents:** 61167
+Additionally, the **Language Identifier Tool** from:  
````
Anticipate this to where the pipeline is introduced

joausaga commented on src/guarascraped/README.md:
```diff
@@ -1,134 +1,132 @@
+### 🏳️ Language
 
-## 📝 Note
+- **Total language score:** 36,023.90
```
What is this?

joausaga commented on src/guarascraped/README.md:
```diff
@@ -1,134 +1,132 @@
 
-Dependencies (spaCy, language identifier models, etc.) must be installed
-beforehand.
+### 📌 Documents
```
The report misses the average proportion of Guarani

Commit:
corrections and improvements in README
